zookeeper-server/src/test/java/org/apache/zookeeper/test/MultiTransactionTest.java

```diff
@@ -787,6 +811,129 @@ public void testTransactionBuilder() throws Exception {
         assertNull(zk.exists("/t2", false));
     }
 
+    @Test
+    public void testMultiGetChildren() throws Exception {
```

Comment:
If it doesn't add too much overhead to unit testing, please try to split this test into 2 test cases.